6/Api/Properties/launchSettings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Api": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "http://localhost:5000"
+  }
+}
```

Comment:
Let's flesh this out a bit with launch url: https://eateam.visualstudio.com/Emailage%20Code/_git/greetings-planet?path=%2Fapi%2FProperties%2FlaunchSettings.json&version=GBmain

Reply:
Added